packages/accounts/src/light-account/e2e-tests/light-account.test.ts

```diff
@@ -195,6 +200,52 @@ describe("Light Account Tests", () => {
     expect(newOwnerViaProvider).not.toBe(oldOwner);
     expect(newOwnerViaProvider).toBe(newOwner);
   }, 100000);
+
+  it("should upgrade a deployed light account to msca successfully", async () => {
```

Comment:
I tested undeployed light account using our gas manager, but now since aa-alchemy optionally depends on aa-accounts, making aa-alchemy a dev dependency of aa-accounts introduces a circular dependency that won't build correctly.